market/retrieval/remoteblockstore/metric.go

```diff
@@ -0,0 +1,123 @@
+package remoteblockstore
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+
+	"github.com/filecoin-project/lotus/metrics"
+)
+
+// Distribution
+var defaultMillisecondsDistribution = view.Distribution(0.01, 0.05, 0.1, 0.3, 0.6, 0.8, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000, 3000, 4000, 5000, 7500, 10000, 20000, 50000, 100000)
```

Comment:
This is way too much imo, prometheus doesn't like that much arity in data, would maybe do 30% of those

Reply:
Dumped most of these.